resources/assets/less/bem/mp-history-game.less

```diff
@@ -115,17 +137,23 @@
 
   &__player-score {
     width: 100%;
+    height: 125px;
```

Comment:
not sure if useful (1)

Reply:
iirc this was to fix the issue with Safari on mobile <https://github.com/ppy/osu-web/pull/1312#issuecomment-315274981>